decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -37,17 +37,19 @@ def serialize
           latitude: proposal.latitude,
           longitude: proposal.longitude,
           state: proposal.state.to_s,
+          state_published_at: proposal.state_published_at,
           reference: proposal.reference,
           answer: ensure_translatable(proposal.answer),
           answered_at: proposal.answered_at,
-          votes: proposal.proposal_votes_count,
+          votes: (proposal.proposal_votes_count unless
```

Comment:
We should extract this (with the spec) on a PR as it is a `type: fix`, as we should backport this. Can you do that please? Thanks 

(you can leave it here too, to prevent git conflicts)

Reply:
As in, create a new PR for the proposal count (as a fix?) and extract the spec for it also? 

I could do that once this PR is ready to merge, so both are added at the same time? What do you think?

Reply:
Extracted fix - https://github.com/decidim/decidim/pull/13739